docs/reference/operators/mobile_package_op_type_support_1.11.md

```diff
@@ -0,0 +1,139 @@
+---
```

Comment:
is it possible to have a top-level page that links to all of these per-version pages?
something that could be referred to from here: https://github.com/microsoft/onnxruntime/blob/d699805e24a2b8b9cd0b86a845c9f816101ecf6d/docs/install/index.md?plain=1#L75
otherwise, that's another line that gets outdated.
one idea is to have another subfolder, e.g., `docs/reference/operators/mobile_package_op_type_support/`

Reply:
The problem is the docs setup only seems to support 2 levels of navigation. I would prefer to have them in a separate subfolder with automatic table of contents (same for the docs on the individual ORT mobile helpers), but if you do that you lose the backlinks. 

@natke is there a way to support more than 2 levels so we're not fighting with the infrastructure to organise things more naturally?

Reply:
Unfortunately, no, not with this site generator. Let's get this one merged and I will have a think about how we can do this better.